FULL_LOCAL_SETUP.md

```diff
@@ -12,7 +12,7 @@ NOTE: If you want to only run the front-end (and rely on a deployed contract and
 
 ### Setup web server
 
-Follow instructions in [web-server/README.md](./web-server/README.md), specifically the `Creating a keypair` section, then the `Start the Webserver` section. (The earlier pieces are for running the web server against )
+Follow instructions in [web-server/README.md](./web-server/README.md), specifically the `Creating a keypair` section, then the `Start the Webserver` section. (The earlier pieces are for running the web server against a contract running on a non-local blockchain)
 
 ### Setup Contract on Polygon Mumbai Testnet
```

Comment:
This is not related to this update, but just below the headline it says 

> You can't run the contract locally with hardhat since it's Polygon and not Ethereum.

I think this is not accurate right? At least I managed to just get everything up and running locally with no issues :-) Just need a EVM compatible blockchain.

Reply:
Good point, this *should* work locally. I'll just remove this extra verbiage.